src/base/requirements/viashpy.yaml

```diff
@@ -1,2 +1,2 @@
 packages:
-  - viashpy==0.7.0
\ No newline at end of file
+  - viashpy==0.8.0
```

Comment:
Is this change needed? Because it might affect other components once merged in

Reply:
yes because of onclass only working with python 3.8 which viash 0.7 doesn't support